### PR TITLE
Updated Loading Screen! (With TOON TIPS)

### PR DESCRIPTION
**Pull request by ghost**
Updated the loading screen to give it a more polished look and it has toon tips!
http://i.imgur.com/MRpdN7W.png

### CODE REVIEW

**Comment by osxnick** [comment]
Looks awesome and slick dude, keep up the good work! :)

**Comment by ghost** [comment]
Thanks! Don't worry,  you will be seeing more from me ;)

-Leroy Wackyboing on Toontown Infinite/Fellowship/Online/Every Server That I Play On Ever I Use That Toon

File: toontown/toontowngui/ToontownLoadingScreen.py [hunk 1]
```diff
@@ -61,8 +63,9 @@ def __init__(self):
         # self.logo.setPos(scale[0], 0, -scale[2])
         self.logo.setPos(0, 0, -scale[2])
         self.toon = None
-
+
```
Comment (judge2020) [inline toontown/toontowngui/ToontownLoadingScreen.py:66]: nice spacing

**Comment by ghost** [comment]
@judge2020 What?

**Comment by DissonancePM** [comment]
@judge2020 that happens in github differences sometimes.

**Comment by ghost** [comment]
@Loudrob Wow. Now I see, haha, Best change ever: delete one line of spaces, add a few more lines of spaces, instant successful game!

File: toontown/toontowngui/ToontownLoadingScreen.py (hunk 1 above)
Comment (ghost) [inline toontown/toontowngui/ToontownLoadingScreen.py:66]: Thanks!

**Comment by osxnick** [comment]
TTF Staff: Merge please<3

**Comment by ghost** [comment]
@xJollibee One thing, i need to make a slight change

**Comment by osxnick** [comment]
To tweak what? @gamesawdrew

**Comment by ghost** [comment]
@xJollibee There is one thing that is a problem, the tips are a bit small, and the wordwrapping is horrible, i will submit a new pull request because i have no idea how to modify one

**Comment by ghost** [comment]
here is the link to the pull request: https://github.com/toontownfellowship/src/pull/79

**Pull request closed without merging**